Filter out of non-Java entries from Felix classpath

enterit opened the pull request:
When non-Java libraries are present in the module dependencies, then the plugin fails to compile such a module. This change fixes the issue.

chilicat commented:
@enterit I added you as a collaborator. Can you merge your changes into the mainline.

enterit commented:
Thank you Daniel! Now I feel like I have more responsibilities :)
Does the code look good? What about the other ScrProcessor implementation? Does it need the same filtering?

enterit committed:
Filter out of non-Java entries from Felix classpath

IDEA allows configuring non-Java libraries (i.e. JS libraries). Felix SCR
Plugin fails during build phase trying to process them.
